src/model/immutable/EditorChangeType.js

```diff
@@ -20,6 +20,7 @@ export type EditorChangeType = (
   'backspace-character' |
   'change-block-type' |
   'change-inline-style' |
+  'change-fragment' |
```

Comment:
This is also confusing. What does it mean to "change" a fragment? I think it's probably fine to have a `'move-block'` change type.